v03_pipeline/lib/tasks/write_family_table.py

```diff
@@ -20,22 +20,18 @@ class WriteFamilyTableTask(BaseWriteTask):
     project_remap_path = luigi.Parameter()
     project_pedigree_path = luigi.Parameter()
     ignore_missing_samples_when_subsetting = luigi.BoolParameter(
-        default=False,
```

Comment:
why did you remove these?

Reply:
`WriteFamilyTable` isn't called as its own task anymore, so the defaults aren't actually going to be used.  I generally prefer to have defaults injected in as few places as possible so that args don't get accidentally dropped along the way.